doc/irmin-pack/design/layered_store.md

```diff
@@ -270,27 +270,27 @@ Tezos node. For this reason, we take the following steps:
   single core.
 - The worker uses `fadvise` to inform the operating system that it
   should preferentially **not** devote resources to caching **worker**
-  disk IO.  The OS caches should instead remain allocated to the main
+  disk I/O. The OS caches should instead remain allocated to the main
   Tezos node. In particular, the worker should not disturb the caches
-  for the main process, when the worker calculates the reachable
+  for the main process when the worker calculates the reachable
   objects for the given commit.
 - With the exception of the calculation of the reachable objects from
   a commit, the worker is **simply copying bytes between files.** This
-  makes the worker very simple, and hopefully makes it easy to
-  optimize performance and ensure correctness of operation.
+  makes the worker very simple and hopefully makes it easy to
+  optimise performance and ensure correctness of operation.
 
-### Crash correctness
+### Crash Correctness
 
 We now describe what happens in the event of a system crash. The
-interesting cases are when the worker is active, or has just
+interesting cases are when the worker is active or has just
 terminated.
 
 If the worker process is active, then it may have created part or all
 of the next object-store and next suffix. However, the control file
 will still point to the current object store and suffix. On restart,
-the main process opens the control file, at which point it knows which
+the main process opens the control file. At this point it knows which
 files are the "real" object-store and suffix (those listed in the
-control file), and can delete any other files it finds (which will be
+control file). The main process can delete any other files it finds (which will be
 the partial files the worker was creating).
 
 If the worker process has just terminated and a crash occurs, then the

```

Comment:
Li 299 doesn't have a complete sentence. Could you clarify please?

Reply:
Hm there's an extra empty line 298 (which needs to be removed) but the sentence continues below. Does the following makes more sense?
> If the worker process terminates just before a crash occurs, then either the current `control` file contains the atomically renamed `control.nnnn` worker's file, or it has been left untouched by the worker. If it was successfully renamed, then the main process will start operating on the new object-store and suffix file created by the worker (and will delete any residual old version of those files).

Reply:
Thank you! I fixed it and (hopefully) clarified two of those sentences. Please ensure they're still accuarte.